config.go

```diff
@@ -15,21 +16,25 @@ type Configuration struct {
 	MongoDBDialTimeout time.Duration
 	UseLocalQueue      bool
 	NumWorkers         int
+	UserManager        gimlet.UserManager
```

Comment:
we can't really put this here... interfaces don't seralize well, and this config object is really about bootstrapping the thing in model.Config